cmudb/env/Dockerfile

```diff
@@ -1,23 +1,60 @@
 # NoisePage officially supports Ubuntu 20.04.
 FROM ubuntu:20.04
-# The default argument for docker run is bash.
-CMD ["bash"]
 
 # Suppress interactive dialog.
 ARG DEBIAN_FRONTEND=noninteractive
 
+# Copy the official PostgreSQL image for configurable env vars.
+# https://hub.docker.com/_/postgres/
+ENV POSTGRES_PASSWORD "terrier"
+ENV POSTGRES_USER "noisepage"
+ENV POSTGRES_DB "noisepage"
+ENV POSTGRES_INITDB_ARGS ""
+ENV POSTGRES_INITDB_WALDIR ""
+ENV POSTGRES_HOST_AUTH_METHOD ""
+
+# PostgreSQL env vars that we customize.
+ENV PGDATA "/pgdata"
+ENV PGPORT "15721"
+# NoisePage env vars.
+# BIN_DIR       :   Destination for PostgreSQL binaries.
+ENV BIN_DIR "/home/terrier/repo/build/bin"
+
 # Install packages.
-RUN apt-get -y update
-RUN apt-get -y install sudo
-COPY ./cmudb/env/packages.sh install-script.sh
-RUN echo y | ./install-script.sh all
-
-# Create a non-sudo user.
-RUN adduser --disabled-password --gecos '' dev \
-    && adduser dev sudo \
+COPY ./cmudb/env/packages.sh /packages.sh
+RUN \
+    apt-get -y update \
+    && apt-get -y install sudo \
+    && echo y | /packages.sh all
+
+# Create a non-sudo user and switch to them.
+# This is because PostgreSQL binaries don't like being
+# run as root, e.g., initdb.
+RUN adduser --disabled-password --gecos '' terrier \
+    && adduser terrier sudo \
     && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
-USER dev
-WORKDIR /home/dev
+USER terrier
```

Comment:
Is there any reason we use terrier as the linux user and not noisepage?

Reply:
terriers are cute

Reply:
More seriously, I thought it might be confusing if everything was named noisepage (the project, the user, the DBMS, the DB).